src/messages/MOSDRepOp.h

```diff
@@ -55,7 +55,7 @@ class MOSDRepOp : public Message {
 
   // piggybacked osd/og state
   eversion_t pg_trim_to;   // primary->replica: trim to here
-  eversion_t pg_trim_rollback_to;   // primary->replica: trim rollback
+  eversion_t pg_roll_forward_to;   // primary->replica: trim rollback
```

Comment:
Yeah, a different patch updated the terminology in PGLog to distinguish between discarding the rollback information and rolling forward.  Operationally there's no difference, but it makes explaining what's going on easier.